### DISCUSSION TITLE
part-ii/2018-136/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#26] [Question comments]
Part II 2018 Riemann Surfaces: Paper 1, Section II, F 

https://questions.tripos.org/part-ii/2018-136/

**Comment by artie2000** [comment]
The answer is $H=V\cong C_2\times C_2$. There's a quick way of doing these, ask in the Part II discord and we'll link it.

**Reply by artie2000** [reply]
~~This is wrong, it's actually $V\cong C_2\times C_2$.~~
I didn't know you can edit comments

**Comment by silkca** [comment]
The first part is all just bookwork describing the space of germs.

The second and third parts are also standard, although I don't recall if they are bookwork - the second is a slightly modified version of the valency theorem, and the third uses the fact that $R \backslash f^{-1}(B)$ is path-connected. Both are results from Algebraic Topology.

The fourth part is standard in that you have to know how to do this for general $f$, and this particular one is not special. There is a fast method of doing these which relates the cycle types of a generating set of $H$ to the lifts through $f$ of simple loops around each branch point, which (combined with some pre-existing knowledge of the transitive subgroups $H<S_{\deg f}$) allows one to determine $H$ exactly for $\deg f$ small enough.
